frontend/src/components/NewShoot/NewShootSelectInfrastructure.vue

```diff
@@ -12,11 +12,12 @@ SPDX-License-Identifier: Apache-2.0
       :class="{ 'select_infra_card_active elevation-4' : infrastructureKind == selectedInfrastructure }"
       @click="selectInfrastructure(infrastructureKind)"
       :key="infrastructureKind"
+      :color="darkMode ? 'grey darken-2' : undefined"
```

Comment:
in dark mode it is hard to tell which infrastructure is selected

Reply:
I worked on the select infrastructure cards. Still not perfect, maybe we can have another look